attack.go

```diff
@@ -145,18 +146,36 @@ func (s *Scanner) DetectAuthMethods(targets []Stream) []Stream {
 }
 
 func (s *Scanner) attackCameraCredentials(target Stream, resChan chan<- Stream) {
+	consecutiveErrors := 0
+	maxConsecutiveErrors := 5 // Stop after 5 consecutive connection failures
```

Comment:
IMO this is unrelated to the issue mentioned in the ticket, but I'm fine with adding this if we use errors instead of booleans to implement it 👍